external/fv3fit/tests/emulation/test_train_microphysics.py

```diff
@@ -177,3 +178,27 @@ def test_TrainConfig_build_loss():
     loss_value, _ = loss(data, data)
     assert 0 == pytest.approx(loss_value.numpy())
     transform.forward.assert_called()
+
+
+def test_TrainConfig_GscondClassesV1():
+    timestep = 1.0
+    # thresholds set to 1e-15 at time of writing tests
+    data = {
```

Comment:
I would consider targeting the "sums to one" at the lower level unit `classes` where that behavior is defined and perhaps just test for presence of the required names here explicitly.